store.go

```diff
@@ -1749,6 +1729,23 @@ func (s *store) Container(id string) (*Container, error) {
 	return rcstore.Get(id)
 }
 
+func (s *store) ContainerLayerID(id string) (string, error) {
+	rcstore, err := s.ContainerStore()
+	if err != nil {
+		return "", err
+	}
+	rcstore.Lock()
+	defer rcstore.Unlock()
+	if modified, err := rcstore.Modified(); modified || err != nil {
+		rcstore.Load()
+	}
+	container, err := rcstore.Get(id)
+	if err != nil {
```

Comment:
I see no code path where it would ever be successful and not return a container, so I think we are all set here.

We call store.Get functions all over and never check the first return value.